### DISCUSSION TITLE
Transparent option problem on floating hints.

### DISCUSSION THREAD

**Discussion by directormac** [catppuccin/nvim#505] [Q&A] [answered]
![image](https://github.com/catppuccin/nvim/assets/5866196/54380658-474c-4f02-9a48-9f752db0fa88)

The black background on hint only shows when using transparent mode.

![image](https://github.com/catppuccin/nvim/assets/5866196/9a059bb3-e021-4979-993c-37df61d981a8)

Working well on non transparent

**Comment by rewhile** [comment, marked as answer]
Because lazyvim set a default values for these you should try adding to `config/options.lua`:

```lua
vim.o.pumblend = 0
vim.o.winblend = 0
```

For more details please see `:h pumblend` and `:h winblend`

**Reply by directormac** [reply]
Thank you so much!!!!!
